### PR TITLE
Codememes

### PR DESCRIPTION
**Pull request by jjzabkar**
Add Codememes game type `GameType.MEMES`.

codememes

## Gameplay Screenshot 

Screen Shot 2022-11-30 at 10 13 21 PM

## Changes

* Add JPG files for `MEMES` game type. Ratio is 1.5:1 width:height for most memes (`assets/memes/*`)
* Refactor all word lists to `services/word-lists` subdirectory (`word-lists.service.ts`)
* Add 28 [twemoji v14](https://twemoji-cheatsheet.vercel.app/) emojis (`emoji-word-list.js`)
* Promote `GameType` to top-level enum (`types.ts`)
* Fix typops, note NodeJS requirements (`README.md`)
* CSS changes:
  * Un-selected tile `border` to `6px` to make pictures and emojis more obvious to spymaster (`game-board.component.scss`), shown in pull request screenshot
  * Grey out and increase opacity for picture cards (`game-board.component.scss`)
* Add _Winter Edition_: combination of word lists for `christmas`, `thanksgiving`, and `halloween`
* Make less common game types (`wordList`s) hidden by default.

## Testing

Ran local `ionic serve`: tested `GameType`s of `WORDS`, `EMOJIS`, `PICTURES`, and `MEMES` as both spymaster and operative.

## TODO ⚠️ ⚠️

`onCreate` seems like it should limit memes to 20 tiles, but tests always show 25.

### CODE REVIEW

**Comment by bacontech** [comment]
I don't know my memes well, this will negatively impact my elo.

**Comment by jjzabkar** [comment]
> I don't know my memes well, this will negatively impact my elo.

@bacontech Correct: You've precisely highlighted the point. 🧑‍🌾 🚜  📈

File: types.ts [hunk 1]
```diff
@@ -123,8 +123,9 @@ export interface Game {
   blueAgents: number;  // remaining blue agents
   redAgents: number;   // remaining red agents
   status: GameStatus;
-  roomId: string;
```
Comment (Matttaylor8910) [inline types.ts:126]: Is `roomId` not used? I believe it is

Comment (Matttaylor8910) [reply types.ts:126]: oh lmao it's just below

**Review by Matttaylor8910 — APPROVED** [review]

**Pull request merged**